x509util/crlcheck/crlcheck.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// crlcheck is a utility to show and check the contents of certificate
+// revocation lists (CRLs).
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/google/certificate-transparency-go/x509"
+	"github.com/google/certificate-transparency-go/x509util"
+)
+
+var caFile = flag.String("ca", "", "CA certificate file")
+var verbose = flag.Bool("verbose", false, "Verbose output")
+
+func addCerts(filename string, pool *x509.CertPool) {
```

Comment:
Is this used?

Reply:
Yes, on lines 61-62.

Reply:
Doh, no, that's the other file.  Removed.